### DISCUSSION TITLE
Where did smg bullets come from?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13795] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

My server is vanilla.
I found 7 smg bullets on my submarine.
Where did smg bullets come from? what to do with it?
SMG bullets cannot be sold, cannot be inserted into a SMG magazine, cannot be deconstructed
where did they come from and what to do with them?

### Reproduction steps

[TheGlassCannonVanilla.zip](https://github.com/FakeFishGames/Barotrauma/files/15048064/TheGlassCannonVanilla.zip)

unpack save
You can find 7 SMG bullets in fabricator shelf.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by NotWendy** [comment]
They appear from SMG magazines sometimes when switching submarines, I think. This bug got "fixed" at least once before.
